source/clear-linux/tutorials/multi-boot/multi-boot.rst

```diff
@@ -0,0 +1,975 @@
+.. _multi-boot:
+
+Multi-boot Clear Linux with other operating systems
+###################################################
+
+Starting with version 16140, |CLOSIA| uses the Systemd-Boot boot loader,
+which does not support multi-booting without manual manipulation. This
+document shows how to modify the boot loader for |CL| to work with other
+operating systems.
+
+The general process to install other operating systems for a
+multi-booting computer is as follows:
+
+#. Install |CL| first with a EFI partition large enough to store the kernels
+   of other operating systems and their initrds in the case of Linux\*
+   distributions.
+
+#. Install the next operating system without creating its own EFI
+   partition.
+
+#. Boot into the newly installed operating system.
+
+#. For Linux distributions, copy its kernel and `initrd` to the |CL| EFI
+   partition. This step is not needed for Windows\*.
+
+#. Add an entry for the newly installed operating system in the
+   Systemd-Boot menu.
+
+#. Make Systemd-Boot the default boot loader.
+
+#. Repeat the previous steps for each additional operating system. Always
+   install |CL| first. Install other operating systems in any order.
+
+.. note::
+   This process is not guaranteed to work with all Linux distributions and
+   all their versions.
+
+The following operating systems were tested on an Intel® NUC6i7KYK with 32GB
+RAM and a 360GB SSD. Table 1 lists the information specific to the
+installation of the tested operating systems.
+
+.. csv-table:: Table 1: OS specific installation information
+   :header: # , OS, Version, Partition Size [1], Swap Size [2], EFI Partition Size [3], Download Link
+
+   1,Clear Linux,16140,50GB,8GB,1GB,https://download.clearlinux.org/image/clear-15870-installer.img.xz
+   2,Windows ,Server 2016,50GB,N/A,Shared with #1,From Microsoft
```

Comment:
Are these "From Microsoft" notes enough? We should providing links for Microsoft, Redhat, and SUSE.

Reply:
No, maintaining those links is a nightmare since each of them change the locations frequently and without notice.

Reply:
Understood.